src/blueprints/tv2/action-factories/tv2-remote-action-factory.ts

```diff
@@ -116,4 +131,108 @@ export class Tv2RemoteActionFactory {
       disableNextInTransition: false
     }
   }
+
+  private createRecallLastRemoteAsNextAction(): Tv2RecallLastRemoteAsNextAction {
+    return {
+      id: 'recall_last_remote_as_next_action',
+      name: 'Recall last Live',
+      description: 'Recalls the last live that has been on air.',
+      type: PartActionType.INSERT_PART_AS_NEXT,
+      metadata: {
+        contentType: Tv2ActionContentType.REMOTE,
+        remoteNumber: '-1', // Is not used for this action.
+        actionSubtype: Tv2ActionSubtype.RECALL_LAST_REMOTE,
+      },
+      data: {
+        partInterface: {} as PartInterface, // Is determined when called.
+        pieceInterfaces: [] as PieceInterface[], // Is determined when called.
+      }
+    }
+  }
+
+  public isRemoteAction(action: Action): action is Tv2RemoteAction {
+    const tv2Action: Tv2Action = action as Tv2Action
```

Comment:
You can just have the parameter be `Tv2Action`. Then you don't need to cast :)

Reply:
Thanks – updated.